### DISCUSSION TITLE
transform cut cube problem

### DISCUSSION THREAD

**Discussion by mqc007** [neperfepx/neper#762] [Q&A] [answered]
Dear all
I wanna make a rock cube with a small fracture inside
so i use the -transform cut code 
but i didnt got the right answer 
here is what i want
![5cbee80a4f0041c79e3ca0b8cf09c35](https://github.com/neperfepx/neper/assets/41913110/0191e4b0-5a96-4270-b323-459d9663342e)
according to the offical webpage 
![1697034357437](https://github.com/neperfepx/neper/assets/41913110/21ba312d-5bb2-455e-9ca0-c4f651fc4633)
i wrote down the coordinates for the fracture and rock cube itself 
here is the code i used:
`neper -T -n 200 -domain "cube(0.06,0.025,0.12)" -transform "cut(cubei(0.02,0.04,0,0.025,0.058,0.068))"  -o frac`
here is the result i got
![1697034207632](https://github.com/neperfepx/neper/assets/41913110/de37a286-bcd5-4855-9d5b-8b99bb123b3a)
it seems i got the part which i truely want to cut 

i am confusing about how use the -transform cut(cube()) code
please tell me how to use this code correctly 
thank you very much 

best wishes

**Comment by rquey** [comment, marked as answer]
`cubei` means that you keep the part of the tessellation located inside the cube.  Unfortunately, `cube` is not available, so it is not possible to do what you want.
